src/clr-angular/deprecations.spec.ts

```diff
@@ -7,4 +7,8 @@
 describe('Deprecations', () => {
   // When we deprecate some code, we should write a test to verify it is still in the bundle
   // and keep track of when it was deprecated, and when we plan to remove it.
+
+  describe('1.0', () => {
+    it('should no longer support clr-checkbox in button-groups');
```

Comment:
This is to help me keep track of it yes. This is a breaking change we cannot avoid right now without working on button groups, and so there isn't much to test since its just removed. Maybe we'll support it again though after 1.0.